config/pmd-test.xml

```diff
@@ -210,6 +210,7 @@
                                  //MethodDeclaration[@Name='verifyOutput']
                       |//ClassOrInterfaceDeclaration[@Image='CommonUtilTest']
                                  //MethodDeclaration[@Name='testClose']
+                      |//ClassOrInterfaceDeclaration[@Image='XdocsUrlTest']
```

Comment:
please see other comment, it is better to avoid new suppressions

Reply:
done~